api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -994,9 +994,14 @@ def close_lid(self) -> None:
 
     @requires_version(2, 21)
     def open_lid(self) -> None:
-        """Close the lid of the Absorbance Reader."""
+        """Open the lid of the Absorbance Reader."""
         self._core.open_lid()
 
+    @requires_version(2, 21)
+    def is_lid_on(self) -> bool:
+        """Return True if the Absorbance Reader's lid is currently closed."""
```

Comment:
Let's Sphinx it up
```suggestion
        """Return ``True`` if the Absorbance Reader's lid is currently closed."""
```
I'm OK with leaving the implication that `False` means it's open, but if false means open or unknown, we should state that.